FLIR/conservator/managers/searchable.py

```diff
@@ -2,6 +2,11 @@
 from FLIR.conservator.paginated_query import PaginatedQuery
 
 
+class AmbiguousIdentifierException(Exception):
+    def __init__(self, identifier):
+        super().__init__(f"Multiple items found for '{identifier}', use ID")
```

Comment:
I think this should go alongside generic `TypeManager`

Reply:
ok